app/views/events/new.html.erb

```diff
@@ -8,6 +8,9 @@
   <%=f.label :description, "Description"%>
   <%=f.text_area :description, class: 'form-control'  %>
 
+  <%=f.label :guests, "Guests"%>
```

Comment:
For Should have date attribute as well.
This is important for upcoming and past events feature

Reply:
done!, the date field it was created but it wasn't in the forms